### ISSUE TITLE
Training Metrics

### ISSUE THREAD

**Issue by Coice** [viitor-ai/viitor-voice#4]
On your huggingface I see you used gigaspeech, libritts-r, etc as your dataset.

In your configs you have a per device batch size of 4 and gradient accumulation of 32.

For your training, how many and which GPU's did you use, and how much time did it take to fully train?

Thanks!

**Comment by MrWaterZhou** [comment]
For the English model, we used 2 A100 GPUs to train for 10 epochs, which takes approximately 3 days.

Some tricks are applied, such as concatenating inputs to reduce padding length. Updates will be made soon.

P.S. The preprocessing workflow will be updated in the near future, with significant changes. After the update, it will support few-shot cloning. Stay tuned!
